interfaces/repo_test.go

```diff
@@ -517,6 +517,180 @@ func (s *RepositorySuite) TestRemoveSlotFailsWhenSlotIsConnected(c *C) {
 	c.Assert(slot, Not(IsNil))
 }
 
+// Tests for Repository.ResolveConnect()
+
+func (s *RepositorySuite) TestResolveConnectExplicit(c *C) {
```

Comment:
These tests are also begging for being refactored into a list, to avoid all the copy & paste and duplication of logic.